Project.toml

```diff
@@ -18,6 +18,7 @@ SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 SparseDiffTools = "47a9eef4-7e08-11e9-0b38-333d64bd3804"
+Static = "aedffcd0-7271-4cad-89d0-dc628f76c6d3"
```

Comment:
Why is this here? We shouldn't rely on Static please.

Reply:
My git is a bit messy. I create a new one and will make a new PR. Shouldn't have static this time.